BootloaderCorePkg/Library/BootloaderLib/BootloaderLib.c

```diff
@@ -357,33 +358,33 @@ GetFeatureCfg (
 }
 
 

```

Comment:
All changes in this file below this line are whitespace changes that I did not make intentionally (line endings?).